homeworks/oleksandr.khodakivskyi_Moo0oony/GamesProject/js/homework.js

```diff
@@ -0,0 +1,58 @@
+function addition(firstValue, secondValue) {
+    return firstValue + secondValue;
+}
+
+function subtraction(firstValue, secondValue) {
+    return firstValue - secondValue;
+}
+
+function multiplication(firstValue, secondValue) {
+    return firstValue * secondValue;
+}
+
+function division(firstValue, secondValue) {
+    return firstValue / secondValue;
+}
+
+function remainder(firstValue, secondValue) {
+    return firstValue % secondValue;
+}
+
+/* function exponentiation(firstValue, secondValue) {
+    return firstValue ** secondValue;
+} */
+
+function calculate(firstValue, secondValue, operation) {
+    let result = 0;
+    const first = Number(firstValue);
+    const second = Number(secondValue);
+    if (first.isNaN || second.isNaN) {
+        alert('Enter a number');
+    }
+    switch (operation) {
+        case '+':
+            result = addition(first, second);
```

Comment:
... you can use the native `+` without your function: `result = first + second`;